rcsbsearchapi/search.py

```diff
@@ -170,22 +172,36 @@ def __xor__(self, other: "Query") -> "Query":
         """Symmetric difference: `a ^ b`"""
         return (self & ~other) | (~self & other)
 
+    # def exec(self, return_type: ReturnType = "entry", rows: int = 10000, return_content_type: List[ReturnContentType] = ["experimental"],
+    # results_verbosity: VerbosityLevel = "compact") -> "Session":
```

Comment:
Can this be cleaned up (and below)? Or were  you planning to add/update these lines later?